deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -39,10 +39,31 @@ These tables list all CDK context configuration values a user can specify for th
 | dpPipelineTemplatePath         | string  | "path/to/config.yaml"                                 | Path to a local Data Prepper pipeline configuration YAML file that Fetch Migration will use to derive source and target cluster endpoints and other settings. Default value is the included template file i.e. [dp_pipeline_template.yaml](dp_pipeline_template.yaml) |
 
 ### Reindex from Snapshot (RFS) Service Options
-| Name                              | Type    | Example                                               | Description                                                                                                                                                  |
-|-----------------------------------|---------|-------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| reindexFromSnapshotServiceEnabled | boolean | true                                                  | Create resources for deploying and configuring the RFS ECS service                                                                                           |
-| sourceClusterEndpoint             | string  | `"https://source-cluster.elb.us-east-1.endpoint.com"` | The endpoint for the source cluster from which RFS will take a snapshot                                                                                      |
+| Name                              | Type    | Example                                                              | Description                                                                                                                              |
+|-----------------------------------|---------|----------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|
+| reindexFromSnapshotServiceEnabled | boolean | true                                                                 | Create resources for deploying and configuring the RFS ECS service                                                                       |
+| reindexFromSnapshotExtraArgs      | string  | "--target-aws-region us-east-1 --target-aws-service-signing-name es" | Extra arguments to provide to the Document Migration command with space separation. See RFS Extra Arguments below for available options. Boolean negation is used, so --no-insecure can be passed to remove the --insecure flag if present. |
+| sourceClusterEndpoint             | string  | `"https://source-cluster.elb.us-east-1.endpoint.com"`                | The endpoint for the source cluster from which RFS will take a snapshot                                                                  |
+
+#### RFS Extra Arguments 
```

Comment:
I'm not sure I would actually document all of this here. Do we already have it documented in an RFS readme? Can we just point to it there? (and if not, can we move this table there?)

We expose extraArgs for a lot of tools and I don't want us to have to maintain documentation for all possible args in multiple places. Additionally, I think we're slightly more likely to confuse people by making them think they have to specify e.g. a snapshot name, when usually we handle it.

Reply:
Will address in followup PR due to github actions flakiness

Reply:
https://github.com/opensearch-project/opensearch-migrations/pull/864